data/topics.yml

```diff
@@ -78,6 +76,8 @@
       url: "paas/how-to-access-environment-variables-inside-dockerfile"
     - title: "What is the difference between the CMD in my Dockerfile and the processes in my Procfile?"
       url: "paas/what-is-difference-between-dockerfile-cmd-and-procfile"
+    - title: "What is the difference between an internal and an external doomain?"
+      url: "paas/what-is-difference-between-internal-external-domain.md"
```

Comment:
Note that the `.md` should not be included here. This is what was causing the failing tests (and would cause a broken link on the live site).